packages/framework/data-object-base/src/lazyLoadedDataObjectFactory.ts

```diff
@@ -63,14 +63,13 @@ export class LazyLoadedDataObjectFactory<T extends LazyLoadedDataObject>
 	): Promise<FluidDataStoreRuntime> {
 		const runtimeClass = mixinRequestHandler(
 			async (request: IRequest, rt: FluidDataStoreRuntime) => {
-				const maybeRouter: FluidObject<IProvideFluidRouter> | undefined =
-					await rt.entryPoint?.get();
+				const maybeRouter: FluidObject<IProvideFluidRouter> = await rt.entryPoint.get();
```

Comment:
Same comment as pure data object factory. Change this to `router` and remove the next assert.